tailwind.config.js

```diff
@@ -6,6 +6,11 @@ module.exports = {
     { pattern: /text-(black|warning|error|success|lavender|chili|papaya)/ },
     { pattern: /text-(primary|secondary|lemon|turquoise|flint|mint|coral|gray|white)-(100|200|300|500|700|900)/ }
   ],
+  theme: {
```

Comment:
the font-face imports live in the projects and not in the plugin, so I would like to discuss adding the `fontFamily` in this local TW config instead of the plugin

Reply:
[pr to add it to the plugin here](https://github.com/lob/tailwind-plugin-lob/pull/39)

Reply:
This approach seems fine to me